reference/api/search.md

```diff
@@ -59,7 +59,7 @@ Query terms enclosed in double quotes are treated as [phrase searches](#query-q)
 
 | field                | Description                    |    type    |
 | -------------------- | ------------------------------ | :--------: |
-| **hits**             | Results of the query           | `[result]` |
+| **hits**             | Results of the query           | `array of objects` |
```

Comment:
Shouldn't you update this for line 150 as well?

Reply:
Absolutely right, thanks for catching that!